cypress/integration/pages/frontPage/tests.js

```diff
@@ -83,6 +85,27 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
               .should('be.visible');
           });
         });
+
+        if (serviceHasIndexAlsos(service)) {
+          it('should contain Index Alsos at a mobile view', () => {
+            cy.viewport('iphone-5');
+            cy.get('section li')
+              .eq(0)
+              .within(() => {
+                cy.get('div div div a')
+                  .eq(0)
+                  .within(() => {
+                    cy.get('span').then($el => {
+                      if ($el.length > 1) {
```

Comment:
what if we have exactly one index also? shouldn't this be a `>=`?

Reply:
The logic here is looking at the anchor tag within the `index-also`. The current logic implemented is that if the `span` is ` > 1` then an SVG will be present. The reason for me writing the condition around this logic was because I couldn't see any other way of detecting if an SVG would be present or not.

Reply:
ahh the first bit makes sense, my bad.

But we also have non-media index alsos, and those shouldn't have a SVG iirc

Reply:
Correct. When the non-media `index-alsos` appear their `span` will `length = 1`. When an `index-also` is media based the `span` will have `length > 1`.

Reply:
ahh okay 👍